docs/cloud/paymaster.mdx

```diff
@@ -0,0 +1,471 @@
+
+### 2. Add Policy Basic Info 
+
+1. Write a descriptive name for your policy
+2. Add a start date for your policy. The current day and time is selected as a default. However, you can set a time in the future to align your 
+policy's start date with a campaign or similar. 
+3. Optionally, add an end date and your policy will automatically lapse on the end date
+
+### 3. Select Chains
+
+All our supported chains will be enabled by default, but it is possible to
+deselect a chain to only sponsor transactions on selected chains
+
+#### Currently Mainnet Supported Chains
+- Ethereum
+- Polygon 
+- Avalanche
+- Mantle 
+- Celo 
+- Optimism 
+- Base 
+- Zora 
+- Gnosis 
+- Metis
+
+#### Currently Supported Testnet Chains 
+- Sepolia
+
+### 4. Add Policy Logic
+
+Policy logic can be updated in one of two ways:
+1. JSON editor
+2. No-code builder (Will be available soon)
+
+There are distinct advantages to this:
+1. You can maintain an auditable history of your policies by storing the JSONs for your policy
+2. You can autogenerate policies for your campaigns since it's a simple JSON schema
+3. The policies can be extremely flexible as you can chain conditional checks for which smart contract to target, which method within the contract, the expected gas, etc.
+
+Once the policy is created, it will become active immediately if its start date has already passed, making it ready for use.
+
+## Sponsoring transactions
+
+Based on [EIP 5792](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-5792.md) and [ERC 7677](https://github.com/ethereum/ERCs/blob/master/ERCS/erc-7677.md), the 
+way to utilize a paymaster is simply providing the paymaster RPC URL. 
+
+There is an `RPC URL` selector in the Paymaster screen to get chain-specific URLs. All the URLs follow the same format with only the chain ID, so you can generate the URL on your dApp based on the active chain.
+
+### Policy Selection
+
+By default, all policies will be validated in succession until one returns true.
+
+## JSON Schema: A Practical Example Walkthrough
+
+### Basics
+The schema itself is a simple object representation of the following
+> Is some key from a user operation equivelant to _(or some other comparision)_  some value?
+
+This is defined in `params` as such:
+```json
+{
+  "key": "some_key",
+  "op": "equals",
+  "value": "some_value"
+}
+```
+
+### Chaining Conditionals
+
+However, where this becomes flexible is the ability to chain different conditionals together. A more concrete example is 
+checking if the User Op targets a smart contract you specify and a specific function within that contract's ABI. Since those fields
+are encoded within a UserOp's `callData`, we will need to use two different methods to access this information.
+1. `callDataMethodEquals`: Verify that the callData targets a specific function
+2. `callDataToEquals`: Verify that the callData targets a specific contract
```

Comment:
```suggestion
### Chaining Conditionals

The flexibility here lies in the ability to chain different conditionals together. For example, you can check if the User Op targets a specific smart contract and a particular function within that contract's ABI. Since these fields are encoded within a UserOp’s `callData`, two different methods are needed to access this information:
1. `callDataMethodEquals`: Verifies that the `callData` targets a specific function
2. `callDataToEquals`: Verifies that the `callData` targets a specific contract
```

Grammar fix